fips.md

```diff
@@ -0,0 +1,80 @@
+# FIPS mode
+
+## What is FIPS?
+
+According to [Wikipedia](https://en.wikipedia.org/wiki/Federal_Information_Processing_Standards) 
+> Federal Information Processing Standards (FIPS) are publicly announced standards developed by the National Institute of Standards and Technology for use in computer systems by non-military American government agencies and government contractors.
+> FIPS standards are issued to establish requirements for various purposes such as ensuring computer security and interoperability and are intended for cases in which suitable industry standards do not already exist.
+
+In short it is a thoroughly tested and verified set of standards which could be used to implement high level of security. In terms of software we are usually speaking specifically about FIPS 140-2. 
+
+ReadonlyREST uses OpenSource [BouncyCastle](https://www.bouncycastle.org) library to provide FIPS 140-2 compliant algorithms.
+
+## Does ReadonlyREST is fully FIPS compliant?
+
+Unfortunately not yet. Currently only SSL transport part uses FIPS compliant algorithms.
+
+## How to enable SSL FIPS compliance
+
+1. Prepare keystore and truststore in BCFKS format which is FIPS compliant. Your existing JKS or PKCS12 keystore could be easily converted to BCFKS. Process is described [in this section](#how-to-convert-jkspkcs12-keystore-files-into-bcfks). 
+> :warning: BCFKS format is supported only when FIPS mode is enabled. It won't be recognised otherwise.
```

Comment:
Why tho?  Is there a reason? Or it's just not implemented?

Reply:
BCFKS format is handled by BouncyCastle provider and this provider is added only when FIPS mode is enabled. I didn't see the reason to include it in non-FIPS mode. Additionally it could be kind of feature because when you provide BCFKS keystore and forget to enable FIPS mode ROR won't start.

Reply:
Fair enough!